tox.ini

```diff
@@ -21,9 +21,14 @@ commands=pdoc3 --html --output-dir docs --force src/hypertrace
 recreate=True
 
 [testenv:test-unit]
+setenv =
+       PYTHONPATH=./src
+
 basepython=python3
 deps = -r{toxinidir}/requirements.txt
        -r{toxinidir}/dev-requirements.txt
 
-commands=pytest -rPx src/hypertrace
+commands=
+       pytest -rPx src/hypertrace
+       pytest -rPx tests/agent
```

Comment:
New integration tests should be added in the Makefile's integration-tests rule, not here.

Reply:
Okay, just to make verify, unit tests are expected to be in the `src` directory alongside library files?
Integrations tests belong under tests/<framework>.

Reply:
That is the pattern @jcchavezs established.

Reply:
Awesome, thanks for the clarification here :)